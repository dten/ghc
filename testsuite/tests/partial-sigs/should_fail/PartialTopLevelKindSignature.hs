{-# LANGUAGE NamedWildCards #-}

module PartialTopLevelKindSignature where

type T :: _w
data T = MkT
