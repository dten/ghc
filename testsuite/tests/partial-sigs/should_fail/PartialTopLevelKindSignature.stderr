PartialTopLevelKindSignature.hs:5:11:
    Wildcard ‘_w’ not allowed
      in the top-level kind signature for T
